bin/oref0-setup.sh

```diff
@@ -221,6 +221,14 @@ if openaps vendor add --path . mmeowlink.vendors.mmeowlink 2>&1 | grep "No modul
     echo Installing latest mmeowlink
     sudo pip install -U mmeowlink || die "Couldn't install mmeowlink"
 fi
+# Install Bluez for BT Tethering
```

Comment:
good, point, but you can get your bt mac ID from your Android phone under settings/about phone/status  and on the iphone settings/general/about